docs/source/configuration/overview.mdx

```diff
@@ -602,7 +602,7 @@ telemetry:
     experimental_apollo_signature_normalization_algorithm: enhanced # Default is legacy
 ```
 
-Once you enable this configuration, operations with enhanced signatures appear with different operation IDs than they did previously in GraphOS Studio.
+Once you enable this configuration, operations with enhanced signatures may appear with different operation IDs than they did previously in GraphOS Studio.
```

Comment:
stephen always made me change "may" to "might" 🤣

Reply:
```suggestion
Once you enable this configuration, operations with enhanced signatures might appear with different operation IDs than they did previously in GraphOS Studio.
```